docs/UserGuide.md

```diff
@@ -63,7 +63,8 @@ Before you continue reading the rest of our user guide, the table below displays
 3. Move the file to the folder you want to use as the _home folder_ for **Trackermon**.
 
 4. Double-click the file to start the app. The layout of Trackermon's [GUI](#glossary) is shown in the [section below](#user-interface).
-
+   * Do note that for Linux OS, you may have to [enable double-click to run JAR files](https://askubuntu.com/a/270175) first! <br><br>
+   
```

Comment:
Good job accounting for Linux user! Maybe you can mention something about Windows and Apple users?

Reply:
Windows and MacOS users should have no issues double clicking the JAR file to execute it. This led me to think that addressing them would be redundant.
What do the rest think?

Reply:
I think there is no need to include Windows and MacOS users.

Reply:
If Mac and Windows have additional setting that require you to enable before allow the user to double-click the jar file. Then we should include Mac and Windows users.

Reply:
Last I checked MacOS supports the double-clicking for execution of JAR files by default.
In that case we should be able to not mention both Windows and MacOS users in the Quick Start as it would be redundant. Thanks everyone for your input!